agent-bootstrap/src/main/java/io/opentelemetry/auto/bootstrap/instrumentation/jdbc/JDBCConnectionUrlParser.java

```diff
@@ -777,14 +783,15 @@ public static DBInfo parse(String connectionUrl, final Properties props) {
       return DEFAULT;
     }
 
-    String baseType = jdbcUrl.substring(0, typeLoc);
-    DBInfo.Builder parsedProps = DEFAULT.toBuilder().type(baseType);
+    String baseSystem = jdbcUrl.substring(0, typeLoc);
+    baseSystem = updateDbSystem(baseSystem);
```

Comment:
something like (see comment below), also probably still want `typeParsers` key to be `type` and not `system`
```suggestion
    String type = jdbcUrl.substring(0, typeLoc); 
    String system = updateDbSystem(type);
```

Reply:
`typeParsers` is using `db.system` as the key though.  I will show you offline.